dandi/tests/test_organize.py

```diff
@@ -249,3 +250,44 @@ def error_link(src, dest):
     monkeypatch.setattr(os, "symlink", succeed_link if sym_success else error_link)
     monkeypatch.setattr(os, "link", succeed_link if hard_success else error_link)
     assert detect_link_type(tmp_path) == result
+
+
+@pytest.mark.parametrize("mode", ["copy", "move"])
+@pytest.mark.parametrize("video_mode", ["copy", "move", None])
+def test_video_organize(video_mode, mode, create_video_nwbfiles, clirunner, tmp_path):
+    dandi_organize_path = create_video_nwbfiles.parent / "dandi_organized"
+    dandi_organize_path.mkdir(parents=True, exist_ok=True)
+    cmd = [
+        "--files-mode",
+        mode,
+        "--rewrite",
+        "external-file",
+        "--external-files-mode",
+        video_mode,
+        "-d",
+        str(dandi_organize_path),
+        str(create_video_nwbfiles),
+    ]
+    video_files_list = [
+        i for i in (create_video_nwbfiles.parent / "video_files").iterdir()
+    ]
+    video_files_organized = []
+    r = clirunner.invoke(organize, cmd)
+    assert r.exit_code == 0
+    for nwbfile_name in dandi_organize_path.glob("**/*.nwb"):
+        vid_folder = nwbfile_name.with_suffix("")
+        assert vid_folder.exists()
+        with NWBHDF5IO(str(nwbfile_name), "r", load_namespaces=True) as io:
+            nwbfile = io.read()
+            # get iamgeseries objects as dict(id=object_id, external_files=[])
+            ext_file_objects = _get_image_series(nwbfile)
+            for ext_file_ob in ext_file_objects:
+                filename = vid_folder.name + f'/{ext_file_ob["id"]}_external_file_'
+                for name in ext_file_ob["external_files"]:
+                    video_files_organized.append(name)
+                    # check if external_file arguments are correctly named according to convention:
+                    assert filename in str(name)
```

Comment:
please make it a bit more specific -- e.g. should `name` end with that filename? 
check that it is not an absolute path (right?)

Reply:
yes, this assertion is more general. I have made it more specific in this commit: https://github.com/dandi/dandi-cli/pull/841/commits/5a49a8a341b9204a46a74a2b951b5b92936a5a5d